lang/cs/Org.Apache.REEF.Driver/Constants.cs

```diff
@@ -32,12 +31,6 @@ public static class Constants
         /// The class hierarchy file from .NET.
         /// </summary>
         [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
-        public const string ClassHierarachyBin = "clrClassHierarchy.bin";
-
-        /// <summary>
-        /// The class hierarchy file from .NET.
-        /// </summary>
-        [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
         public const string ClassHierarchyBin = "clrClassHierarchy.bin";
```

Comment:
Please remove other deprecated fields in `Constants.cs`, except for `NullHandler` for which we need to remove deprecation (it is used in a method which was marked for deletion but was made internal instead, so we have to keep it).